migration/migration.go

```diff
@@ -649,14 +653,79 @@ func createOrUpdateWorkItemLinkType(ctx context.Context, linkCatRepo *link.GormW
 	return nil
 }
 
+// createCategory creates category in db
+func createOrUpdateSingleCategory(ctx context.Context, categoryRepo category.Repository, categoryID *uuid.UUID, categoryName string) error {
```

Comment:
Btw. this function is called `createOrUpdate...` but it only tries to create a category and it doesn't try to update it in case it already exists. I think you're only calling this function in one place. If so, then remove this function and inline the one line of creating and the second line of error checking.